src/main/java/org/owasp/esapi/logging/java/JavaLogFactory.java

```diff
@@ -43,10 +43,15 @@
 
 /**
  * LogFactory implementation which creates JAVA supporting Loggers.
- *
- * This implementation requires that a file named 'esapi-java-logging.properties' exists on the classpath.
- * <br>
- * A default file implementation is available in the configuration jar on GitHub under the 'Releases'
+ * <br><br>
+ * Options for customizing this configuration (in recommended order)
+ * <ol>
+ * <li>Consider using the <i>SLF4JLogFactory</i> with a java-logging implementation.</li>
+ * <li>Configure java LogManager environment properties as defined by the <i>java.util.logging.LogManager</i> API</li>
+ * <li>Overwrite the esapi-java-logging.properties file with the desired logging configurations. <br>A default file implementation is available in the configuration jar on GitHub under the 'Releases'</li>
+ * <li>Specifically set the system property for LogManager in the custom-code solution to prevent ESAPI JavaLogFactory from overriding project configuration</li>
```

Comment:
I'm not clear what you mean by this. More specifically, what is different about this than about the comment at line 50 (other than those are system properties and not environment variables).

Reply:
@kwwall, I think you're referring to line 52 vs 50?
>[50] Configure java LogManager environment properties as defined by the <i>java.util.logging.LogManager</i> API

and

> [52] Specifically set the system property for LogManager in the custom-code solution to prevent ESAPI JavaLogFactory from overriding project configuration


My intent is that line 50 would be adding the appropriate `-D` to the application initialization.
Line 52 is more referring to adding the call to `System.setProperty` in the application client code.

The end state should be about the same, as long as the `System.setProperty` invocation happens before ESAPI starts up.  That race condition is why it is lower on the recommended order.  It *might* work, but someone needs to understand the application initialization order.